### PR TITLE
mention grape_logging in documentation

### PR DESCRIPTION
**Pull request by aserafin**
Adding request logging with parameters, error codes etc was always the first thing after starting new grape project for me so I though I would make a gem out of it and make it easier for the other developers.   Putting it in the readme would help spread the word!

[grape_logging](https://github.com/aserafin/grape_logging)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1589,6 +1589,8 @@ class API < Grape::API
 end
 ```
 
+For similar to Rails request logging try [grape_logging](https://github.com/aserafin/grape_logging) gem.
```
Comment (dblock) [inline README.md]: try "the" ..., no?

Comment (aserafin) [inline README.md]: true, fixed

**Comment by dblock** [comment]
Can you squash these, please?

**Comment by aserafin** [comment]
commits squashed

**Comment by dblock** [comment]
Merged.

**Pull request merged**